### DISCUSSION TITLE
怎样在chatflow里将聊天历史赋值到变量里

### DISCUSSION THREAD

**Discussion by nietao2** [langgenius/dify#3317] [General]
我想在code节点里面处理一下聊天历史，但是我看了一下现在的几个节点，只有llm节点可以设置记忆，请问有没有办法在其他节点里通过变量拿到聊天历史？

**Comment by crazywoola** [comment]
Not for now. I will talk to the PM later.

**Reply by ianZzzzzz** [reply]
The process variables in the workflow are very important to us. Although the 0.6.15 version has environment variables, it is not possible to assign values ​​to these variables during the workflow. In customer service and other businesses, we use multiple classifications to determine the business type. This classification only needs to be done once. We hope to add variables and combine them with the if judgment module to skip repeated function blocks.

**Reply by iamjoel** [reply]
This feature is currently under development right now. Coming soon~

**Reply by koking0** [reply]
> This feature is currently under development right now. Coming soon~

Is this feature available? How do I use it?
